src/site/content/en/blog/youtube-media-capabilities/index.md

```diff
@@ -0,0 +1,84 @@
+authors:
+  - beaufortfrancois
+date: 2019-06-05
+hero: hero.jpeg
+hero_position: top
+alt: Youtube application screengrab
+description: |
+  In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase
+  in MTBR with only a 0.4% decrease in average resolution of videos served.
+tags:
+  - post
+  - media
+  - ux
+  - case-study
+---
+
+In an experiment with the Media Capabilities API, YouTube saw a 7.1% increase in
+MTBR with only a 0.4% decrease in average resolution of videos served.
+
+{% Aside 'key-term' %}
+MTBR (Mean Time Between Rebuffers) is total play time divided by the number of
+rebuffering events.
+{% endAside %}
+
+## The Problem
+
+Typically, media sites will have several variants that they can present users,
+encoded in different frame rates, resolutions, and codecs. Until recently, while
+choosing which variant to play for an individual user, web developers had to
+rely solely on `isTypeSupported()` or `canPlayType()` to determine whether each
+variant could be played in that user’s browser. While this told the developer
+whether media could be played at all, it didn’t provide an indication of
+playback quality, such as whether the media playback would experience frame
+drops or rapidly drain the device’s battery. In the absence of this signal,
+developers either had to create their own heuristics, or just assume that if a
+device could playback a codec/resolution combination, it could do so smoothly
+and power efficiently. For users with less capable devices, this often led to a
+very poor experience.
+
+## The Solution
+
+The [Media Capabilities](https://wicg.github.io/media-capabilities/) API allows
+websites to get more information about the client’s video decode performance and
+make an informed decision about which codec and resolution to deliver to the
+user. Specifically, the API provides the developer with an estimate of
+smoothness and power efficiency of a particular codec and resolution
+combination. This allows the developer to avoid scenarios where the client is
+likely to have a poor playback experience.
+
+In Chrome, the Media Capabilities API uses metrics from previous playbacks to
+predict whether future playbacks in the same codec and at the same resolution
+will be smoothly decoded.
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="./maxime-rossignol-266384-unsplash.jpg" alt="Computer motherboard near graphics card">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 1 — Computer motherboard near graphics card
+  </figcaption>
+</figure>
+
+## YouTube Case Study
+
+YouTube used the [Media
+Capabilities](https://wicg.github.io/media-capabilities/) API to prevent their
+adaptive bitrate algorithm from automatically selecting resolutions that a
+device could not playback smoothly.
+
+Users who were part of the experimental group collectively saw less frequent
+rebuffers (the meantime between rebuffers, or MTBR, increased by 7.1%) while the
```

Comment:
```suggestion
rebuffers (the mean time between rebuffers, or MTBR, increased by 7.1%) while the
```
BTW, I think this is a better way to define the term: the definition occurs inline, proximal to its first use, so readers don't have to momentarily wonder what MTBR is.

Reply:
Done.